data/remnant/remnant missions.txt

```diff
@@ -3445,3 +3445,274 @@ event "remnant: pantica void sprites"
+			choice
+				`	"So, what is our next step?"`
+					goto labnext
+				`	"Seeker?" you ask.`
+			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is both a title that I aspire to earn and one I do not feel that I deserve."`
+			choice
+				`	"Okay. So what next?"`
+					goto labnext
+				`	"I thought all Remnant were spacefarers?"`
+			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a rotation of a month or two of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`
+			choice
+				`	"Okay, let's get on with this."`
+					goto labnext
+				`	"Years drifting in the void?"`
+			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment.`
+			label labnext
+			branch jumpdrive
+				has "Jump Drive" >= 1
+			`	"That is neither here nor there, however. Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "That shouldn't be too hard. Our early warning system indicates that there are several incoming Korath fleets that should be here in the next day or two. Any one of them should have a Jump Drive you can take. Meet me in the Starport once you have a Jump Drive."`
+			label jumpdrive
+			`	""That is neither here nor there, however.." She pauses and checks her commlink. "I just need to take care of a few things, then I will meet you in the spaceport to embark to Sagittarius A*."`
+
+
+
+event "remnant: lab availability"
+	outfitter "Remnant"
+		add "Research Laboratory"
+
+
+
+mission "Remnant: Expanded Horizons Astral 2"
+	name "Research expedition to Sagittarius A*"
+	description "Travel to Sagittarius A* and then return to <destination>."
+	blocked "You need at least 1 free bunk to take this mission."
+	passengers 1
+	waypoint "Sagittarius A*"
+	to offer
+		has "Remnant: Expanded Horizons Astral 1: done"
+		require "Jump Drive" >= 1
+		require "Research Laboratory" >= 1
+	to complete
+		require "Research Laboratory" >= 1
```

Comment:
Also not a thing. #2494

Reply:
requirement for jump drive and research lab moved to `on offer` section.